ui/modal/create_password_modal.go

```diff
@@ -230,6 +229,8 @@ func (cm *CreatePasswordModal) Handle() {
 		}
 	}
 
+	cm.modal.SetDisabled(cm.isLoading)
```

Comment:
Move this to inside the `cm.SetLoading` method so whenever `cm.isLoading` is modified, this is also done. Also want to point out that `cm.isLoading` is never set to false once it's set to true - is this right?

Reply:
I also see a confusing `cm.isEnabled` property. Looking at the code, looks like that should be a `cm.canSubmit` property? It's also never used outside of the `Handle` method, so I'm sure it can be a function-scope variable that is given the appropriate value at the beginning of the `Handle` method.

Reply:
> Move this to inside the `cm.SetLoading` method so whenever `cm.isLoading` is modified, this is also done. Also want to point out that `cm.isLoading` is never set to false once it's set to true - is this right?

This is set from the page calling create_password_modal using the `SetLoading()`.